test/helper/Appium_test.js

```diff
@@ -319,7 +348,7 @@ describe('Appium', function () {
         "//android.widget.TextView[@resource-id = 'io.selendroid.testapp:id/gesture_type_text_view']")
       let vy = yield app.grabTextFrom("//android.widget.TextView[@resource-id = 'io.selendroid.testapp:id/text_view3']")
       assert.equal(type, 'FLICK');
-      assert.equal(vy, 'vx: -12000.0 pps');
+      assert.equal(vy, 'vx: -21000.0 pps');
```

Comment:
this value is relates with device resolution, so maybe it will fail in saucelabs